src/dex/maverick-v2/maverick-v2.ts

```diff
@@ -97,41 +104,38 @@ export class MaverickV2 extends SimpleExchange implements IDex<MaverickV2Data> {
     return null;
   }
 
+  async getEventPools(srcToken: Token, destToken: Token) {
+    srcToken = this.dexHelper.config.wrapETH(srcToken);
+    destToken = this.dexHelper.config.wrapETH(destToken);
```

Comment:
@Wertusser I think we need to add const before `srcToken` and `destToken`

Reply:
fixed